tests/test_outlier_detection/test_auto_outliers.py

```diff
@@ -155,6 +782,251 @@ def test_decide_outliers(self):
         expected_df = self.create_expected_df()
```

Comment:
Checked the `TestDecideOutliers` too even though it's not part of the PR. Looks good.